coral-component-accordion/src/scripts/AccordionItem.js

```diff
@@ -99,7 +99,15 @@ class AccordionItem extends BaseComponent(HTMLElement) {
     return this._selected || false;
   }
   set selected(value) {
-    this._selected = this.hasAttribute('disabled') ? false : transform.booleanAttr(value);
+    let _value = transform.booleanAttr(value);
+    let _selected = this.hasAttribute('disabled') ? false : _value;
+
+    if(this._selected === _selected) {
+      // do nothing if value not changed
```

Comment:
This comment is redundant. I think we can safely remove it from everywhere and not lose any information.